src/base/bittorrent/session.cpp

```diff
@@ -156,10 +156,9 @@ namespace
         return QString::fromUtf8(str.data(), static_cast<int>(str.size()));
     }
 
-    template <typename Entry>
-    QSet<QString> entryListToSetImpl(const Entry &entry)
+    QSet<QString> toTagsSet(const lt::bdecode_node &entry)
```

Comment:
Sorry, I didn't get your previous answer. The function name is now incorrect.
It just load strings from bdecoded list into QSet.
Maybe you were confused by the name of the existing variable inside (`const QString tag`)? Yes, it turns out that it is also incorrect. This function has a broader meaning, just in this particular case (in the code that uses it) it is used to load a list of tags.

Reply:
>It just load strings from bdecoded list into QSet.

Really? what do you think this line `if (Session::isValidTag(tag))` and the below lines do? Just see the full function.

Reply:
Well, you could tell right away that the point is beyond that git diff... Forget it.
But I don't like it anyway. Since it is not conversion-only function I would name it `loadTags` or something like it.
Also isn't qWarning redundant here?

Reply:
I'll inline it since it is only used at one place.